lib/Conversion/FIRRTLToHW/LowerToHW.cpp

```diff
@@ -4724,8 +4721,28 @@ LogicalResult FIRRTLLowering::visitPrintfLike(T op, StringAttr outputFile) {
     }
   }
 
+  result = StringAttr::get(loc->getContext(), formatString);
+  return success();
+}
+
+// Printf/FPrintf is a macro op that lowers to an sv.ifdef.procedural, an sv.if,
+// and an sv.fwrite all nested together.
+template <class T>
+LogicalResult FIRRTLLowering::visitPrintfLike(
+    T op, llvm::PointerIntPair<StringAttr, 1> outputFileAndRequiresSubst) {
+  auto clock = getLoweredNonClockValue(op.getClock());
+  auto cond = getLoweredValue(op.getCond());
```

Comment:
This template is almost hinting that there is a missing interface for "command" statements. That is a discussion for a later time...